portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,30 +55,32 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         PreparedQuery results = datastore.prepare(query);
-        
+
+        String text = getParameter(request, "text", "");  
+        String first = getParameter(request, "first-name", "");   
+        String last = getParameter(request, "last-name", "");  
+      
         for (Entity entity : results.asIterable()) {
             long id = entity.getKey().getId();
-            String comment = (String) entity.getProperty("comment");
+            String comment = (String) entity.getProperty("text");
             long timestamp = (long) entity.getProperty("timestamp");
-            String firstname = (String) entity.getProperty("first name");
-            String lastname = (String) entity.getProperty("last name");
+            String firstname = (String) entity.getProperty("first-name");
+            String lastname = (String) entity.getProperty("last-name");
 
             Comment NewComment = new Comment(id, comment, firstname, lastname, timestamp);
+            System.out.print("New comments: ");
+            System.out.print(NewComment);
             comments.add(NewComment);
+            System.out.print("comments: ");
+            System.out.print(comments);
         }
-    
-    String text = getParameter(request, "text-input", "");   
-    String first = getParameter(request, "first-name", "");   
-    String last = getParameter(request, "last-name", "");   
-    String[] words = text.split("\\s*,\\s*");
-    long timestamp = System.currentTimeMillis(); 
 
     Entity taskEntity = new Entity("Comments");
-    taskEntity.setProperty("comment", text);
-    taskEntity.setProperty("timestamp", timestamp); 
-    taskEntity.setProperty("first name", first);
-    taskEntity.setProperty("last name", last);
-
+    taskEntity.setProperty("text", text);
+    mycomments.add(text);
```

Comment:
Are these just getting saved into memory? I see the field on the class but I think these should get looked up each time.

Reply:
deleted mycomments list that was just for testing